modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -4212,6 +4212,10 @@
 		"issueKey": "LPD-53012",
 		"to": "fetchCommercePriceEntryByExternalReferenceCode(param#1#, param#0#)"
 	},
+	{
+		"from": "new InfoItemFormVariation(String paramName, InfoLocalizedValue<String> paramName)",
+		"issueKey": "LPD-54064"
```

Comment:
@RenanUmV please change to the task LPD: LPD-8024

Reply:
Sorry, I forgot to change that